cds/modules/deposit/api.py

```diff
@@ -415,36 +418,16 @@ def has_minted_doi(self):
         return is_local_doi(self['doi']) if self.is_published() else False
 
 
-def project_resolver(project_id):
-    """Get records from PIDs."""
-    pid = PersistentIdentifier.query.filter_by(
-        pid_value=project_id
-    ).one().object_uuid
-    return Project.get_record(pid)
-
-
-def cds_resolver(ids):
-    """Get records from PIDs."""
-    pids = [p.object_uuid for p in PersistentIdentifier.query.filter(
-        PersistentIdentifier.pid_value.in_(ids)).all()]
-    return CDSDeposit.get_records(pids)
-
-
-def video_resolver(ids):
-    """Get records from PIDs."""
-    pids = [p.object_uuid for p in PersistentIdentifier.query.filter(
-        PersistentIdentifier.pid_value.in_(ids)).all()]
-    return Video.get_records(pids)
-
-
+# TODO move inside Video class
 def video_build_url(video_id):
     """Build video url."""
-    return url_for('invenio_deposit_rest.video_item', pid_value=video_id)
+    return 'https://cds.cern.ch/api/deposits/video/{0}'.format(str(video_id))
 
 
+# TODO move inside Video class
 def record_build_url(video_id):
     """Build video url."""
-    return url_for('invenio_records_rest.recid_item', pid_value=str(video_id))
+    return 'https://cds.cern.ch/api/record/{0}'.format(str(video_id))
```

Comment:
If I correctly remember why I did like this, I'd say because perhaps we can call the function also from `app` and `celery_app`.